web/Makefile

```
@@ -0,0 +1,52 @@
+DIST=$(CURDIR)/dist
+SRC=$(CURDIR)/src
+IMAGES=$(CURDIR)/images
+VENV=$(CURDIR)/.venv
+BUILD=$(CURDIR)/.build
+VARIANTS_DOT=$(SRC)/variants.dot
+
+.PHONY: all prod images dev dev-images clean update-variants-diagram
```

checkmake (0.2.2)

[warning] Missing required phony target "test"

(minphony)